lib/bots/versionbot.ts

```diff
@@ -414,6 +414,7 @@ export class VersionBot extends ProcBot {
         const head = event.cookedEvent.data.pull_request.head;
         const owner = head.repo.owner.login;
         const name = head.repo.name;
+        const author = event.cookedEvent.data.sender.login;
```

Comment:
Is this the author of the whole PR, or the author of the most recent commit which triggered this update?
(and I wonder if it should ping both, if they're different people?)